src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -289,6 +289,18 @@ public function theStoreAlsoOperatesInLocale(LocaleInterface $locale): void
         $this->channelManager->flush();
     }
 
+    /**
+     * @Given the store has :taxCalculationStrategy tax calculation strategy configured
```

Comment:
```suggestion
     * @Given the store uses the :taxCalculationStrategy tax calculation strategy
```

Reply:
You forgot to remove the `configured` at the end 🍻

Reply:
Fixed 😅